docs/UserGuide.adoc

```diff
@@ -103,6 +74,128 @@ automatically loaded into the program when it is opened.
 * START, END - a time parameter, in the following format: HH:mm
 * DATE - a date parameter, in the following format: dd-MM-yyyy
 
+=== Dashboard
+Home page.
+
+=== Student View
+
+This view will show you the list of all your students. It will show you information such
+as student name, matriculation number, attendance, ratings you give the student
+(such as for participation), any notes you have on the student and contact details
+(email and telegram id).
+
+=== Session View
+
+This view will contain the upcoming tasks and statistics.
+
+=== TSS View
+
+This will show you all your claimable hours up till now in the same format as that
+used in the TSS claim files.
```

Comment:
Should be in features not usage

Reply:
Help check line spacing? Some headers not showing properly.

Check out the doc view next to the 3 dots to see what I mean.